packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -763,4 +764,48 @@ describe('PerformanceMonitoring', function() {
     performanceMonitoring.processAPICalls('invoke', task3)
     expect(tr.ended).toBe(true)
   })
+
+  it('should send span context destination details to apm-server', done => {
+    const transactionService = serviceFactory.getService('TransactionService')
+
+    const tr = transactionService.startTransaction('with-context', 'custom', {})
+    const data = {
+      url: 'http://localhost:3000/b/c',
+      method: 'GET',
+      target: {
+        status: 200
+      }
+    }
+    const xhrSpan = tr.startSpan(`GET ${data.url}`, 'external')
+    xhrSpan.end(null, data)
+
+    const resourceUrl = 'http://example.com'
+    const rtData = {
+      url: resourceUrl,
+      entry: resourceEntries.filter(({ name }) => name === resourceUrl)[0]
+    }
+    const rtSpan = tr.startSpan(rtData.url, 'resource')
+    rtSpan.end(null, rtData)
+
+    configService.events.observe(TRANSACTION_END, () => {
+      expect(tr.spans.length).toBe(2)
+      const [span1, span2] = tr.spans
+      expect(span1.context.destination).toBeDefined()
+      expect(span2.context.destination).toBeDefined()
+
+      const payload = performanceMonitoring.convertTransactionsToServerModel([
+        tr
+      ])
```

Comment:
Maybe add checks here that the destination is added to the payload!

Reply:
There is already a check here right? https://github.com/elastic/apm-agent-rum-js/pull/539/files#diff-4f75686bfc54bd212244f355ce6f93f5R793-R794